gui-apps/sddm-gentoo-config/sddm-gentoo-config-0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Gentoo defaults for SDDM (Simple Desktop Display Manager)"
+HOMEPAGE="https://github.com/sddm/sddm"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~x86"
+IUSE="X"
+
+RDEPEND="
+	X? ( x11-base/xorg-server )
+	!X? ( dev-libs/weston )
```

Comment:
can this instead be turned into || (weston kwin) ? that way there is no unnecessary dependency on weston if kwin is installed